guava-tests/test/com/google/common/io/LineBufferTest.java

```diff
@@ -121,6 +127,13 @@ private static List<String> readUsingReader(String input, int chunk, boolean asR
     return lines;
   }
 
+  private static Stream<String> readUsingReaderGetLines(String input, int chunk, boolean asReader) {
+      Readable readable =
+          asReader ? getChunkedReader(input, chunk) : getChunkedReadable(input, chunk);
```

Comment:
When looking through the code base, this is the way they split the ternary operator
```suggestion
      Readable readable = asReader 
          ? getChunkedReader(input, chunk) 
          : getChunkedReadable(input, chunk);
```

Reply:
I referred this function `readUsingReader`.